presto-delta/src/test/java/com/facebook/presto/delta/IncrementalUpdateQueriesTest.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.delta;
+
+import com.facebook.presto.Session;
+import com.facebook.presto.common.type.TimeZoneKey;
+import com.facebook.presto.hive.HivePlugin;
+import com.facebook.presto.testing.MaterializedResult;
+import com.facebook.presto.testing.MaterializedRow;
+import com.facebook.presto.testing.QueryRunner;
+import com.facebook.presto.tests.DistributedQueryRunner;
+import com.facebook.presto.tpch.TpchPlugin;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+import java.util.Map;
+
+import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
+import static com.facebook.presto.testing.assertions.Assert.assertEquals;
+import static java.lang.String.format;
+import static java.util.Locale.US;
+import static org.testng.Assert.assertNull;
+import static org.testng.Assert.assertTrue;
+
+public class IncrementalUpdateQueriesTest
+        extends AbstractDeltaDistributedQueryTestBase
+{
+    private final String version = "delta_v3";
+    private final String controlTableName = "deltatbl-partition-prune";
+    private final String targetTableName = controlTableName + "-incremental";
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        QueryRunner queryRunner = createDeltaQueryRunner(ImmutableMap.of(
+                "experimental.pushdown-subfields-enabled", "true",
+                "experimental.pushdown-dereference-enabled", "true"));
+        registerDeltaTableInHMS(queryRunner,
+                version + FileSystems.getDefault().getSeparator() + targetTableName,
+                version + FileSystems.getDefault().getSeparator() + targetTableName);
```

Comment:
let's use Path instead? 
```suggestion
                Paths.get(version).resolve(targetTableName).toString(),
                Paths.get(version).resolve(targetTableName).toString());
```